Bmx.Idp.Okta/State/OktaAccountState.cs

```diff
@@ -0,0 +1,19 @@
+﻿using System.Linq;
+using Bmx.Core.State;
+using Bmx.Idp.Okta.Models;
+
+namespace Bmx.Idp.Okta.State {
+	public class OktaAccountState : IAccountState {
+		public OktaAccountState( OktaApp[] oktaApps, string accountType ) {
+			OktaApps = oktaApps;
+			AccountType = accountType;
+		}
+
+		public string[] Accounts {
+			get => OktaApps.Where( app => app.AppName == AccountType ).Select( app => app.Label ).ToArray();
```

Comment:
Minor: we could do this once in the constructor instead of every time we call the getter.